agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/BytecodeUtil.java

```diff
@@ -300,7 +341,7 @@ void trackDependency(
         @Nullable Date timestamp,
         String name,
         String id,
-        String resultCode,
+        String duration,
```

Comment:
this doesn't seem right. i think you intended to rename `Long totalMillis` to `Long duraiton` (line 345)?

Reply:
ooops!! indeed that is what I meant to do